js/logo.js

```diff
@@ -8820,9 +8825,19 @@ function Logo () {
                 } else {
                     var cblk1 = that.blocks.blockList[blk].connections[1];
                     var cblk2 = that.blocks.blockList[blk].connections[2];
-                    var a = that.parseArg(that, turtle, cblk1, blk, receivedArg);
+                    var r = that.blocks.blockList[cblk1].value;
                     var b = that.parseArg(that, turtle, cblk2, blk, receivedArg);
-                    that.blocks.blockList[blk].value = that._doMultiply(a, b);
+                    if (r == 'previous' || r == 'next') { 
```

Comment:
Shouldn't these be testing for _('next') and _('previous') as well as 'next' and 'previous'?